### PR TITLE
Bugfix/300 home performance

### PR DESCRIPTION
**Pull request by lucasrlt**
This branch fixes the performance issue on the main screen noticed on lower quality phones.
The previous implementation of the full screen in home activity used a nested scroll view, which forced the load of the entire FullExperienceFragment's recycler view.

This PR refactors the home activity to use only one recycler view that combines the header fragments + the full experiences list. 

You can check the performance improvement by running it on your phone

Also, two minor AR fixes are made: 
- Scale the cards up after last Friday's review
- Fix a bug that makes the ar augmented images not recognized after clicking on the details button

Any comments are appreciated :smile:

**Ready to review**

### CODE REVIEW

**Comment by qtipee** [comment]
It is fluid on my phone (but my-phone-fluid-like which is perfect) ; really good job :)

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,93 +1,14 @@
...
-            android:nestedScrollingEnabled="true"
-            app:layout_constraintTop_toBottomOf="@+id/fragment_experiences_nearby_list"
-            tools:layout="@layout/fragment_experiences_list" />
-    </androidx.constraintlayout.widget.ConstraintLayout>
-
-
-</androidx.core.widget.NestedScrollView>
\ No newline at end of file
+        tools:layout_editor_absoluteX="1dp"
+        tools:layout_editor_absoluteY="1dp" />
```
Comment (ToufiPF) [inline app/src/main/res/layout/activity_main.xml:13]: Is this wanted, or just leftover debug?

File: app/src/main/java/com/pindex/main/adapters/HomeHeaderAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        arrayListOf(SearchButtonFragment(), NearbyExperienceFragment())
+
+    @Override
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
+        // Every item is just a standard frame layout that will be replaced by the inflated fragment.
+        var view =
+            LayoutInflater.from(parent.context).inflate(R.layout.activity_home_header_item, parent, false)
+        var viewHolder = ViewHolder(view)
+
+        return viewHolder;
```
Comment (ToufiPF) [inline app/src/main/java/com/pindex/main/adapters/HomeHeaderAdapter.kt:41]: Look at dat ';' so java 😆

**Review by ToufiPF — COMMENTED** [review]
Nice job, just a few remarks cf. below

File: app/src/main/java/com/pindex/main/adapters/HomeHeaderAdapter.kt (hunk 2 above)
Comment (lucasrlt) [reply app/src/main/java/com/pindex/main/adapters/HomeHeaderAdapter.kt]: Aaaaahhh you caught me :laughing:

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (lucasrlt) [reply app/src/main/res/layout/activity_main.xml]: Definitely leftover

**Review by ToufiPF — APPROVED** [review]

**Review by ToufiPF — APPROVED** [review]

**Review by amomo25 — APPROVED** [review]

**Pull request merged**